README.md

```diff
@@ -1,2 +1,418 @@
+
+### All together
+```js
+// Main.js
+import { Cmd } from 'mangojuice-core';
+import * as SearchForm from './SearchForm';
+import * as SearchResults from './SearchResults';
+
+export const createModel = () => ({
+  form: SearchForm.createModel(),
+  results: SearchResults.createModel()
+});
+export const Logic = {
+  name: 'Main',
+
+  config({ nest }) {
+    return {
+      children: {
+        form: nest(this.HandleSearchForm(), SearchForm.Logic),
+        results: nest(null, SearchResults.Logic),
+      }
+    }
+  },
+
+  @Cmd.batch
+  HandleSearchForm({ model }, cmd) {
+    if (cmd.is(SearchForm.Logic.Search.Before)) {
+      return SearchResults.Logic.Search(model.form.query).model(model.results);
+    }
+  }
+};
+export const View = ({ model }) => (
+  <div>
+    <SearchForm.View model={model.form} />
+    <SearchResilts.View model={model.results} />
+  </div>
+)
+```
+The block contains the form, the results block and ties them together.
+
+**`const createModel = () =>`** as usual makes the initial block's model, but it also creates models for children blocks using their own `createModel` functions.
+
+**`config({ nest })`** helps to define what logic should be associated with what model. In the example above we are saying that the block have two child blocks.
+
+**`form: nest(this.HandleSearchForm(), SearchForm.Logic),`** not only defines that `SearchForm.Logic` will be associated with the `form` model's field, but also set a handler command that will be executed before and after each command from `SearchForm` block. The handler will be also called for every nested block.
+
+**`if (cmd.is(SearchForm.Logic.Search.Before))`** catch `Search` command from `SearchForm` block before the execution. There is also `SearchForm.Logic.Search.After` to handle the command after execution.
+
+**`SearchResults.Logic.Search(model.form.query).model(model.results)`** creates a command associated with some exact model. It means that the command will be executed with some exact model passed in a first argument. By default all commands returned by some command of `Main` block associated with `Main`'s model. But in the example we want to trigger a command of a child block, and `.model(model.results)` helps to do it.
+
+**`<SearchForm.View model={model.form} />`** shows View of `SearchForm` for a model `form`.
+
+As you can see, to nest one block to another you have to nest all three parts of a block separately: model, logic, view. It is a bit overhead, but it makes nesting very flexible. For example, you can use a Logic of one block and compelely replace a View of this block to something more suitable in some concrete situation. Or you can create a model for a child block in some specific way.
```

Comment:
Model, Logic, and View (uppercase)...
It is a bit **of** overhead....
'a Logic' => 'the Logic'
typo: 'comelely' => completely